### ISSUE TITLE
voxel and produced mesh are slighlty off

### ISSUE THREAD

**Issue by afabri** [CGAL/cgal#530]
When making a tetrahedral mesh from segmented voxel data the mesh is 1/2 a voxel off.
It should be enough to shift  the image.

**Comment by lrineau** [comment]
The following patch, not tested, should work:
```diff
--- a/Mesh_3/demo/Mesh_3/Scene_segmented_image_item.cpp
+++ b/Mesh_3/demo/Mesh_3/Scene_segmented_image_item.cpp
@@ -305,7 +305,14 @@ fill_buffer_data()
       }
     }
   }

+  const GLfloat half_vx = data_.vx() / 2;
+  const GLfloat half_vy = data_.vy() / 2;
+  const GLfloat half_vz = data_.vz() / 2;
+  for(std::size_t i = 0, end = vertices_.size(); i < end; i + =3) {
+    vertices_[i] -= half_vx;
+    vertices_[i+1] -= half_vy;
+    vertices_[i+2] -= half_vz;
+  }
   create_arrays();
 }
 ```

**Comment by lrineau** [comment]
Fixed by #534.
